_data/hacks.yml

```diff
@@ -19,6 +19,9 @@
        - verint
        - sas
        - mlh
+  buttons:
+      - text: That's TV Report
```

Comment:
I am gonna be a nitpick here and say that most people don't know what That's TV is, and so the button won't make much sense to them.

Reply:
Yes that's a fair point. What would you suggest in terms of a name change? I thought it would be good to mention that that the event was reported by an external party/the media instead of internally by the school or GUTS. I think it might be worthwhile to keep the organisation name, in case we get reported on by different organisations in the future but open to other options. Maybe something like 'Media Coverage' or 'Video Footage'?